### ISSUE TITLE
scripts - Digitally sign the init.ps1 PowerShell file as well

### ISSUE THREAD

**Issue by benjmor** [chocolatey/choco#1665]
Although the Chocolatey binaries are signed, the PowerShell scripts (those of type .ps1 or .psm1) are not signed. This causes blocks in environments that use script whitelisting.

One example is in choco\nuget\chocolatey\tools\init.ps1. It's a short PowerShell file, but it can still trigger alerts/blocks since it is unsigned.

**Comment by ferventcoder** [comment]
@benjmor actually you might take another look - we digitally sign the binaries AND PowerShell scripts on release.

**Comment by ferventcoder** [comment]
If there is one that is not being signed, that is a bug that will need to be fixed.

**Comment by ferventcoder** [comment]
Let's call it a bug - we don't sign this script as it is not used - 

https://github.com/chocolatey/choco/blob/ea4b155ad4f6911c003fcf30792f85abca480f05/.build.custom/codeSign.step#L124

**Comment by ferventcoder** [comment]
It's the only one that we don't sign. Curious how it is tripping things up as it is not used at all?

**Comment by ferventcoder** [comment]
@benjmor ping.

**Comment by benjmor** [comment]
I have pretty poor visibility since I'm just looking at a list of file executions and don't have local access or knowledge of the workflow. I'm seeing it called from a process that looks a lot like the file 'server.cmd' in this repository: https://gist.github.com/Iristyle/effcc874e567cd1798f9217848e280ec

...which I believe calls into this repo?
https://github.com/puppetlabs/puppetlabs-powershell/blob/master/lib/puppet_x/templates/powershell/init_ps.ps1

So, in short, I don't know how chocolatey gets dragged into this. But something is doing it, at least in our environment.

**Comment by ferventcoder** [comment]
This will be in 0.10.12.
